### DISCUSSION TITLE
Error: Cannot find module

### DISCUSSION THREAD

**Discussion by ipsita4** [outline/outline#3693] [General]
on running  start I am getting this error

> Executing task: npm run start <

> outline@0.64.3 start        
> node ./build/server/index.js

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'C:\Users\Dell\Desktop\outline\outline\build\server\index.js'  
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)        
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command npm run start" terminated with exit code: 1.

Terminal will be reused by tasks, press any key to close it.

Can anyone help me with this?

**Comment by TheNightmanCodeth** [comment]
Sorry to resurrect this but I ran into this as well. The docs don't mention it but you need to run `yarn build` before running `yarn start`. Pretty obvious but just in case someone else gets stumped here that's the "fix"
